src/utils/XRPL/account.ts

```diff
@@ -0,0 +1,63 @@
+import * as xrpl from "xrpl";
+import { Core } from "./core";
+import config from "../../../config";
+import { Wallet } from "xrpl";
+
+export interface IBalance {
+    value: string;
+    currency: string;
+}
+
+export class Account {
+    private readonly walletCold: Wallet;
+    private readonly walletHot: Wallet;
+
+    constructor(walletCold: Wallet, walletHot: Wallet) {
+        this.walletCold = walletCold;
+        this.walletHot = walletHot;
+    }
+
+    // Getter for walletCold
```

Comment:
comments should be reserved only for complicated parts

Reply:
Ok, I removed them.